e2e/test_operator_ha_upgrades/operator_ha_upgrade_test.go

```diff
@@ -474,7 +472,9 @@ var _ = Describe("Operator HA Upgrades", Label("e2e", "pr"), func() {
 				fixtures.PodsSelector(operatorPrimarySatellitePods.Items),
 			}, "20")
 
-			upgradeAndVerify(fdbCluster, targetVersion)
+			Expect(fdbCluster.UpgradeCluster(targetVersion, false)).NotTo(HaveOccurred())
```

Comment:
What is this change about? "UpgradeCluster()" and "checkVersion()" are still getting called, which is what "upgradeAndVerify()" is doing too?

Reply:
The difference here is that we are not waiting until all clusters are fully reconciled. Waiting until the cluster is fully reconciled can take a long time and result in the test suite exceeding the time limits.

Reply:
Maybe we should let a small set of tests wait for all clusters to fully reconcile (by giving them more time to run), in order to find any issues related to that.

Reply:
That sounds like a good idea. We can update some of the tests to do that (or we create a dedicated suite for that).